lib/airbrake-ruby/async_sender.rb

```diff
@@ -99,5 +101,16 @@ def spawn_worker
         end
       end
     end
+
+    def will_not_deliver(notice)
+      backtrace = notice[:errors][0][:backtrace].map do |line|
+        "#{line[:file]}:#{line[:line]} in `#{line[:function]}`"
```

Comment:
Should be `` `#{line[:function]}' `` (no backtick in the end) if you want to make it resemble a Ruby stack frame.